gulpfile.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const gulp = require('gulp')
+const multiaddr = require('multiaddr')
+const Node = require('./src').Node
+const Peer = require('peer-info')
+
+let node
+
+gulp.task('libnode:start', (done) => {
+  node = new Node()
+  const mh = multiaddr('/ip4/127.0.0.1/tcp/9090/websockets')
+  const peer = new Peer()
+  peer.multiaddr.add(mh)
+
+  node.swarm.handle(peer, '/echo/1.0.0', (conn) => {
```

Comment:
This call doesn't take any peerInfo object https://github.com/diasdavid/js-libp2p-swarm/blob/master/src/index.js#L304